gapipy/models/base.py

```diff
@@ -126,7 +126,7 @@ def _set_resource_field(self, field, value):
             for v in value:
                 stub.append(self._model_cls(field)(v, stub=True))
         elif value:
-            stub = self._model_cls(field)(value, stub=True)
+            stub = self._model_cls(field)(value, self._client, stub=True)
         setattr(self, field, stub)
 
     def _set_resource_collection_field(self, field, value):
```

Comment:
I don't _think_ it's super disruptive. Resources need configuration to save, which is what the `Client` defines. I do think there _could_ be a single global `Client` (config), but language would have to be moved to the `Query` instead, since that is state that varies per-call. I think that's where the fundamental problem lies. Language is a query-time setting, not global, whereas almost all other Client settings are global.